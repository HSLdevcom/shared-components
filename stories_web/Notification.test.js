import initStoryshots from '@storybook/addon-storyshots';

initStoryshots({
  storyKindRegex: /^Notification$/
});
