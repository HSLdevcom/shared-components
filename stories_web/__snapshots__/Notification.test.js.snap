// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Notification Error 1`] = `
<div>
  <div
    className="undefined sc-ktHwxA gCnVKo"
  >
    <div
      className="content sc-bdVaJa bDWFJH"
    >
      <div
        className="icon sc-bdVaJa bDWFJH"
      />
      <div
        className="message sc-bdVaJa bDWFJH"
      />
    </div>
  </div>
  <button
    className="sc-htpNat kIHcXL"
    onClick={[Function]}
  >
    Error title, close
  </button>
  <button
    className="sc-htpNat kIHcXL"
    onClick={[Function]}
  >
    Error title + text, close
  </button>
  <button
    className="sc-htpNat kIHcXL"
    onClick={[Function]}
  >
    Success title + text, timeout
  </button>
  <button
    className="sc-htpNat kIHcXL"
    onClick={[Function]}
  >
    Success text, custom fast timeout
  </button>
  <button
    className="sc-htpNat kIHcXL"
    onClick={[Function]}
  >
    Error titleLink, close
  </button>
  <button
    className="sc-htpNat kIHcXL"
    onClick={[Function]}
  >
    Error titleLink and textLink, close
  </button>
  <button
    className="sc-htpNat kIHcXL"
    onClick={[Function]}
  >
    Success everything notification, close
  </button>
  <button
    className="sc-htpNat kIHcXL"
    onClick={[Function]}
  >
    Neutral, close
  </button>
  <button
    className="sc-htpNat kIHcXL"
    onClick={[Function]}
  >
    Netural everything notification, close
  </button>
</div>
`;
